package.json

```diff
@@ -11,6 +11,7 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/pickers": "^3.3.10",
+    "@mui/x-data-grid": "^4.0.1",
```

Comment:
It looks like you added this package but don't use it anywhere. If you don't plan on using it, please remove it.